core/src/avm2/globals/xml.rs

```diff
@@ -759,3 +759,29 @@ pub fn replace<'gc>(
     // 10. Return x
     Ok(xml.into())
 }
+
+pub fn set_notification<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    avm2_stub_method!(activation, "XML", "setNotification");
+    let xml = this.as_xml_object().unwrap();
+    let node = xml.node();
+    let fun = args[0];
```

Comment:
This should probably be
```
let fun = args.try_get_object(activation, 0);
```
which reduces the following code to 
```
    node.set_notification(
        fun.as_function_object(),
        activation.context.gc_context,
    );
```

Reply:
Nice, thanks.